cypress/features/regression/multiActionButton.feature

```diff
@@ -3,56 +3,16 @@ Feature: Multi Action Button default component
 
   @positive
   Scenario Outline: Change Multi Action Button text to <text>
-    When I open default "Multi Action Button" component in noIFrame with "multiActionButton" json from "commonComponents" using "<nameOfObject>" object name
+    When I open default "Multi Action Button Test" component in noIFrame with "multiActionButton" json from "commonComponents" using "<nameOfObject>" object name
     Then Multi Action Button text on preview is set to <text>
     Examples:
       | text                         | nameOfObject         |
       | mp150ú¿¡üßä                  | textOtherLanguage    |
       | !@#$%^*()_+-=~[];:.,?{}&"'<> | textSpecialCharacter |
 
-  @positive
-  Scenario Outline: Change buttonType property of Multi Action Button component to <buttonType>
-    When I open default "Multi Action Button" component in noIFrame with "multiActionButton" json from "commonComponents" using "<nameOfObject>" object name
-    Then Multi Action Button has "<background-color>" background-color
-      And Multi Action Button border color is "<border-color>" border-color
```

Comment:
remove this step from `multi-action-button-steps.js` file

Reply:
can't remove those steps - are still used in other TC